sdk/python/feast/repo_operations.py

```diff
@@ -136,6 +139,12 @@ def apply_total(repo_config: RepoConfig, repo_path: Path):
         ),
         feature_views=repo.feature_views,
     )
+    infer_event_timestamp_column_for_data_sources(
+        [view.input for view in repo.feature_views]
+    )
+    for view in repo.feature_views:
+        view.infer_features_from_input_source()
+
```

Comment:
Is there an ordering requirement that these be put up here? I think that in lines 162-166 [below](https://github.com/feast-dev/feast/pull/1646/files#diff-9922445cc092f7a2f8885c771cb4d15c4c21a84bcbc0513ba76009036ddbccf5R162-R166), we validate if the data sources are even supported (and in https://github.com/feast-dev/feast/pull/1627, whether the BQ tables even exist). I'm happy to change the order in #1627 later, but wanted to ask now about whether order matters.

Reply:
good call